target_actionnetwork/client.py

```diff
@@ -43,8 +43,8 @@ def _request(
     
     @property
     def base_url(self) -> str:
-        return self.config.get("uri")
+        return "https://actionnetwork.org/api/v2/"
```

Comment:
Why did you change it?

Reply:
i first put here and it was one of my first tickets here in hotglue. after work more weeks here i realize that our pattern is to have it usually here hardcoded, the exceptions is to put in the config. what it is not this case